### PR TITLE
[EI Analytics] Migration of EI Analytics from DAS to SP

### PR DESCRIPTION
**Pull request by RolandHewage**
Analytics profile of WSO2 EI 6.4.0 is based on WSO2 Stream Processor.
When upgrading from WSO2 EI 6.x.x to 6.4.0, you are moving from an instance of WSO2 DAS to WSO2 Stream Processor. EI Analytics table migration is an indirect process where we need to convert the Analytics tables into RDBMS tables by running spark scripts using the CarbonJDBC provider packed with DAS. 

- A shell script & a batch script is given to the user to create EI Analytics tables with indexes prior to the migration process. 
- Indexes cannot be created in RDBMS tables by running spark scripts using the CarbonJDBC provider packed with DAS. So EI Analytics tables are created by running a simple shell script or batch script prior to the migration of EI Analytics data. 
- User can specify the relevant RDBMS database & the EI Analytics tables are created in the specified database by running the script. 
- Then the EI Analytics data are migrated by running the spark script using the CarbonJDBC provider packed with DAS.

-  [Data Mapping EI DAS and EI SP](https://docs.google.com/spreadsheets/d/1edFDMUfPOTsKJPhFDa2oc9Y00FWCCFQmZgLURPRImM0/edit#gid=0)
- [Migration of EI Analytics from DAS to SP](https://docs.google.com/document/d/1lwsJX89yI3w1EWjO1HojF_NVTKBAn1mOSg6Ir5Wscm4/edit)

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

JDK version (1.8)
Operating System (Ubuntu 18.04)
Databases (mysql/postgresql/mssql/oracle)

### CODE REVIEW

File: distribution/src/analytics/migration/migration-EI6.x.x-6.4.0/resources/migEIAnalytics.bat [hunk 1]
```diff
@@ -0,0 +1,80 @@
+@echo off
+
+REM ---------------------------------------------------------------------------
+REM   Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Nirothipan) [inline distribution/src/analytics/migration/migration-EI6.x.x-6.4.0/resources/migEIAnalytics.bat]: can we update the year please

Comment (RolandHewage) [reply distribution/src/analytics/migration/migration-EI6.x.x-6.4.0/resources/migEIAnalytics.bat]: Fixed in 39d9d9c

File: distribution/src/analytics/migration/migration-EI6.x.x-6.4.0/resources/mig-ei-analytics/pom.xml [hunk 2]
```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sdkottegoda) [inline distribution/src/analytics/migration/migration-EI6.x.x-6.4.0/resources/mig-ei-analytics/pom.xml]: Shall we add the licence header?

Comment (RolandHewage) [reply distribution/src/analytics/migration/migration-EI6.x.x-6.4.0/resources/mig-ei-analytics/pom.xml]: Fixed in 46c3488

File: distribution/src/analytics/migration/migration-EI6.x.x-6.4.0/resources/mig-ei-analytics/pom.xml [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>one-jar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <!--&lt;!&ndash; Make this jar executable &ndash;&gt;-->
```
Comment (sdkottegoda) [inline distribution/src/analytics/migration/migration-EI6.x.x-6.4.0/resources/mig-ei-analytics/pom.xml]: Shall we remove the commented configuraiton?

Comment (RolandHewage) [reply distribution/src/analytics/migration/migration-EI6.x.x-6.4.0/resources/mig-ei-analytics/pom.xml]: Fixed in 700800a

**Pull request merged**